commandapi-core/src/main/java/dev/jorel/commandapi/arguments/CommandResult.java

```diff
@@ -0,0 +1,20 @@
+package dev.jorel.commandapi.arguments;
+
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+
+import java.util.Arrays;
+
+public record CommandResult(Command command, String[] args) {
+	public void execute(CommandSender sender) {
+		command.execute(sender, command.getLabel(), args);
+	}
+
+	@Override
+	public boolean equals(Object o) {
```

Comment:
CodeFactor complains that this record doesn't have a corresponding `hashCode()` method:

> The contract of `equals()` and `hashCode()` requires that equal objects have the same hashCode. Therefore, whenever you override `equals()` you must override `hashCode()` to ensure that your class can be used in hash-based collections.

We'll need a `hashCode()` method for this record.